tapcfg/config.go

```diff
@@ -439,6 +428,17 @@ func LoadConfig(interceptor signal.Interceptor) (*Config, btclog.Logger, error)
 		return nil, nil, err
 	}
 
+	// Initialize logging at the default logging level.
```

Comment:
I think the log call on line 424 and 427 will now go to `stdout` only, since we haven't configured the logger yet? Maybe we should explicitly print those to `stderr` instead?

Reply:
Yeah it'll just go to `stdout`, tested this locally. Anything logged in `ValidateConfig` will also go to `stderr` as well. Basically what happened is that during the refactor for `tapd` stuff, we made some assumptions about the way things are set up. I think we can do another refactor here (as robyn hinted below) to fix things, but then we'd also need to change the `litd` creates the daemons.